error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/OptionalRules.java

```diff
@@ -220,6 +222,22 @@ T after(Optional<T> o1, Optional<T> o2) {
     }
   }
 
+  /**
+   * Prefer {@link Optional#orElseGet(Supplier)} over {@link Optional#orElse(Object)} if the given
+   * value is not a compile-time constant.
+   */
+  abstract static class OrElseToOrElseGet<T> {
```

Comment:
```suggestion
  static final class OrElseToOrElseGet<T> {
```
(This one is also suggested when running `./apply-error-prone-suggestions.sh`.)